WeatherApp/WeatherHandler.swift

```diff
@@ -93,4 +158,33 @@ class WeatherHandler {
         }
         task.resume()
     }
+    
+    func getWeatherForCities(completion: @escaping (_ result: Bool) -> Void) {
```

Comment:
you should only call completion block once, after all the cities data was updated.